org.standardnotes.standardnotes.metainfo.xml

```diff
@@ -33,6 +33,9 @@
   </categories>
 
   <description>
+    <p><em>(Note: This release is built by a 3rd party from official Standard Notes releases. Standard Notes is 
```

Comment:
Sorry for the late response!

Please move this to the last line of the description, and rephrase it like `NOTE: This release is not verified by, affiliated with, or supported by Standard Notes Ltd.` which is a standard statement as appeared in https://flathub.org/apps/details/com.getpostman.Postman

https://flathub.org/apps/details/com.getpostman.Postman